cli/flox/src/commands/init/python.rs

```diff
@@ -950,69 +928,48 @@ mod tests {
 
         assert_eq!(pyproject.unwrap(), PyProject {
             provided_python_version: ProvidedVersion::Compatible {
-                requested: Some(">=3.8".to_string()),
-                compatible: ProvidedPackage::new("python3", vec!["python39"], "3.9.18"),
+                requested: Some("<=3.10".to_string()),
+                compatible: ProvidedPackage::new("python3", vec!["python3"], PYTHON_310_VERSION),
             },
         });
     }
 
-    /// ProvidedVersion::Incompatible should be returned for requires-python = "1"
+    /// ProvidedVersion::Compatible should be returned for requires-python with space.
```

Comment:
git messed up the diff here :/

Reply:
Yeah, I noted in the commit that it's probably easier view side-by-side. I tried splitting the move and the rename to separate commits but it made no difference.